datamade/erikson-edi
# allow overlays to show on different domains and subdomains

## Overview

Updates the logic in the map display to allow for the currently selected domain or subdomain to stay selected when adding circle overlays on top.

Creates two convenience functions `EDI.getDomainTitle` and `EDI.isDomain` to keep the code a bit more DRY and more reliable save state.

Closes #110 

### Demo

![edi1](https://user-images.githubusercontent.com/919583/93382732-29aff880-f828-11ea-8718-59487cf95945.gif)

## Testing Instructions

Open up the map interface. Select different combinations of domains/subdomains and overlays. In all cases, the underlying choropleth map should not change. The legend titles and cutoffs for the domains and subdomains should also remain the same.
